src/components/BasketballCourt/TopKeyArea.tsx

```diff
@@ -10,18 +10,18 @@ interface TopKeyAreaProps {
 }
 
 const TopKeyArea: React.FC<TopKeyAreaProps> = ({ startPoint }) => {
-  const { keyAreaWidth, threePointLineToCourtEdgeLenth, threePointLineRadius, circleRadius } =
+  const { keyAreaWidth, threePointLineToCourtEdgeLength, threePointLineRadius, circleRadius } =
     useStoreSelector((state) => state.courtSize);
   const startPointX = startPoint.X + keyAreaWidth;
-  const startPointY = startPoint.Y + (threePointLineToCourtEdgeLenth + threePointLineRadius);
+  const startPointY = startPoint.Y + (threePointLineToCourtEdgeLength + threePointLineRadius);
 
   const selectedColor = useStoreSelector((state) => state.courtColor.selectedColor);
   const topKeyAreaColor = useStoreSelector(
     (state) => state.tile.find((tile) => tile.location.includes("topKeyArea"))?.color
   );
   const dispatch = useDispatch();
   const handleColorChange = () => {
-    if (!selectedColor) return;
+    if (selectedColor === "none") return;
     dispatch(changeTileColor({ selectedColor, location: "topKeyArea" }));
   };
```

Comment:
Why don't you use if(selectedColor) then dispatch? Did I get it wrong?

Reply:
coz the fill color for color.svg will be black if there is no value on initial selectColor state. But the fill color will be transparent when initial state is "none"~

Reply:
> Why don't you use if(selectedColor) then dispatch? Did I get it wrong?

Coz seletedColor will always have a value, but we need it stop working when state is "none".

Reply:
Will be great if you could write some comments for someone who don't know